doc/includes/reveal/examples_reveal_basic_rendered.html

```diff
@@ -4,5 +4,5 @@
   <h2 id="modalTitle">Awesome. I have it.</h2>
   <p class="lead">Your couch.  It is mine.</p>
   <p>I'm a cool paragraph that lives inside of an even cooler modal. Wins!</p>
-  <a class="close-reveal-modal" aria-label="Close">&#215;</a>
+  <a class="close-reveal-modal" aria-label="Close" role="button" tabindex="0">&#215;</a>
```

Comment:
This is not really true for anchors without href. They are skipped otherwise at least in some browsers.